x/liquidstake/keeper/liquidstake.go

```diff
@@ -719,6 +720,54 @@ func (k Keeper) LiquidUnbond(
 	return completionTime, returnAmount, ubd, nil
 }
 
+// DivideByWeight divide the input value by the ratio of the param weight of the liquid validator and return it with crumb
+// which is may occur while dividing according to the weight of active liquid validators by decimal error.
+func (k Keeper) DivideByWeight(
```

Comment:
This is the important test. All the others are just accommodated to the new LSM staking params.